### PR TITLE
Fixing precision and refactoring

### PR DESCRIPTION
**Pull request by MuTsunTsai**
This PR contains two things:
1. Fix the precision of isInside_sweet again to fix folding errors in additional samples.
2. Fix all minor coding issues suggested by VS Code Java linting, including:
- Remove unused import.
- Comment out unused variables and private methods (I keep them as comments just in case).
- Add default clauses to switch.
- Close JarFile to prevent potential memory leak.
- Add missing generic parameters.

### CODE REVIEW

**Review by qurben — COMMENTED** [review]
Can you remove the commented out code?

File: oriedita/src/main/java/oriedita/editor/export/Obj.java [hunk 1]
```diff
@@ -59,7 +59,7 @@ public static Save importFile(File file) throws IOException {
                 if (str.equals("v")) {
                     d1 = Double.parseDouble(tk.nextToken());
                     d2 = Double.parseDouble(tk.nextToken());
-                    d3 = Double.parseDouble(tk.nextToken());
+                    // d3 = Double.parseDouble(tk.nextToken());
```
[the remark this answers is not in the document]
Comment (MuTsunTsai) [reply oriedita/src/main/java/oriedita/editor/export/Obj.java:62]: How about if I just invoke `tk.nextToken()` without the `parseDouble` etc.?

File: oriedita/src/main/java/oriedita/editor/service/ButtonService.java [hunk 2]
```diff
@@ -207,8 +207,8 @@ public void actionPerformed(ActionEvent e) {
             button.doClick();
         }
 
-        public AbstractButton getButton() {
-            return button;
-        }
+        // public AbstractButton getButton() {
```
[the remark this answers is not in the document]
Comment (MuTsunTsai) [reply oriedita/src/main/java/oriedita/editor/service/ButtonService.java:210]: OK sure.

File: oriedita/src/main/java/oriedita/editor/swing/dialog/ExportDialog.java [hunk 3]
```diff
@@ -170,7 +170,7 @@ private void onClose() {
         final JPanel panel3 = new JPanel();
         panel3.setLayout(new GridLayoutManager(1, 2, new Insets(0, 0, 0, 0), -1, -1));
         contentPane.add(panel3, new GridConstraints(0, 0, 1, 1, GridConstraints.ANCHOR_CENTER, GridConstraints.FILL_BOTH, GridConstraints.SIZEPOLICY_CAN_SHRINK | GridConstraints.SIZEPOLICY_CAN_GROW, GridConstraints.SIZEPOLICY_CAN_SHRINK | GridConstraints.SIZEPOLICY_CAN_GROW, null, null, null, 0, false));
-        list1 = new JList();
+        list1 = new JList<>();
```
Comment (qurben) [inline oriedita/src/main/java/oriedita/editor/swing/dialog/ExportDialog.java:173]: This is generated code btw, so i'm not sure if these changes will persist.

Comment (MuTsunTsai) [reply oriedita/src/main/java/oriedita/editor/swing/dialog/ExportDialog.java:173]: Huh. Well I've tried.

**Pull request merged**